seedlet/functions.php

```diff
@@ -106,20 +106,11 @@ function seedlet_setup() {
 		// Add support for editor styles.
 		add_theme_support( 'editor-styles' );
 
-		$editor_stylesheet_path = './assets/css/style-editor.css';
-
-		// Note, the is_IE global variable is defined by WordPress and is used
-		// to detect if the current browser is internet explorer.
-		global $is_IE;
-		if ( $is_IE ) {
-			$editor_stylesheet_path = './assets/css/ie-editor.css';
```

Comment:
Not sure why we are still checking this here because we don't use or have IE specific stylesheets anymore.